rai/client.go

```diff
@@ -266,12 +266,20 @@ func marshal(item interface{}) (io.Reader, error) {
 // Unmarshal the JSON object from the given response body.
 func unmarshal(rsp *http.Response, result interface{}) error {
 	data, err := pareseHttpResponse(rsp)
+
 	if err != nil {
 		return nil
 	}
 
 	switch data.(type) {
 
+	// Cancel request response body is nil
+	case nil:
```

Comment:
I see what you mean about this being inconsistent with other responses. maybe we should return `{}` from the transaction service… cc @salamehsameera

Reply:
thanks @salamehsameera!